.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Describe your changes
+
+## Issue ticket if applicable
+
+<!-- Fix: https://github.com/jupyter/docker-stacks/issues/0 -->
+
+## Checklist (especially for first-time contributors)
+
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added thorough tests
+- [ ] I will try not to use force-push to make the review process easier for reviewers
+
+<!-- markdownlint-disable-file MD041 -->
```

Comment:
This is a better way to hide these comments, so people do not see these comments when they start reading this file while filling the PR template.